Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.7-alpine
+
+RUN apk add --no-cache build-base gcc bash cmake git
+
+RUN gem install bundler -v "~>1.0" && gem install bundler jekyll
+
+WORKDIR /site
+
+RUN gem install github-pages 
+
+RUN echo 'source "https://rubygems.org"' >> Gemfile
+RUN echo 'gem "github-pages", "~> 213", :group => :jekyll_plugins' >> Gemfile
+
+ENV PAGES_REPO_NWO="usr/repo"
```

Comment:
I think this is something, that we should change. In the docker context we're not in a git repo, so there's no way for jekyll to figure out the repo name. So it would throw an error and refuse to start.

But there is a third option: The repo name can also be configured in the `_config.yml`, maybe we should do this?

Reply:
Or, less invasive: We provide the whole user's project folder as a volume (see below), instead of just the docs folder. This would include the ".git" folder and *should* work, too.

Reply:
I would be in favor of a solution that required 0 changes to the codebase. So I like the idea of copying in the entire codebase even though it produces a fat Docker image.

Reply:
Now I just added the `.git` folder as a second volume. The fifth option, better forget about the other ones :) That seems to work really fine.
@IMax153 By using volumes we're not even producing large docker images. Files are not copied at build time, but linked at runtime of the docker image. In this setting the user does not have to rebuild and restart with docker to see a change reflected in the docs.